Gemfile

```diff
@@ -32,6 +32,7 @@ gem "sass-rails", "~> 5.0"
 gem "uglifier", ">= 1.3.0"
 
 gem "bourbon"
+gem "matrix"
 
 # See https://github.com/rails/execjs#readme for more supported runtimes
```

Comment:
Need to add this gem in ruby 3.1 explicitly.

Read more about it here: https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/
and an stackoverflow answer https://stackoverflow.com/questions/71991514/rails-require-cannot-load-such-file-matrix